src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -390,7 +370,7 @@ public void verifyUserLoggedIn(final String userName) {
       } else {
         WebElement avatar = wait.forElementVisible(By.cssSelector(LOGGED_IN_USER_SELECTOR_OASIS));
         String loggedInUserName = avatar.getAttribute("alt");
-        if (!loggedInUserName.equals(userName)) {
+        if (!loggedInUserName.equals(userName) && !loggedInUserName.equals(userName + " avatar")) {
```

Comment:
I thought so myself, but it `{username} avatar` still appears on www.wikia.com for authenticated user - so I need to check this condition, since `verifyUserLoggedIn` is not aware of the page context